plugins/modules/source_control/gitlab/gitlab_group_variable.py

```diff
@@ -151,6 +151,7 @@ class GitlabGroupVariables(object):
 
     def __init__(self, module, gitlab_instance):
         self.repo = gitlab_instance
+        self.group.per_page = 100
```

Comment:
This is wrong, it should be 
```suggestion
        self.repo.per_page = 100
```